docker-images/linux-nodejs-x64/Dockerfile

```diff
@@ -1,15 +1,19 @@
 FROM 123124136734.dkr.ecr.us-east-1.amazonaws.com/aws-common-runtime/ubuntu-16.04:x64
 
-ENV NODE_VERSION="10.1.0"
+ENV NODE_VERSION="10"
 
 ###############################################################################
 # NodeJS
 ###############################################################################
 RUN set -ex \
-    && curl "https://nodejs.org/download/release/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" -o node-v$NODE_VERSION-linux-x64.tar.gz \
-    && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
-    && rm "node-v$NODE_VERSION-linux-x64.tar.gz" \
-    && ln -s /usr/local/bin/node /usr/local/bin/nodejs \
+    && curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.0/install.sh | bash \
+    && export NVM_DIR="$HOME/.nvm" \
+    && [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" \
+    && command -v nvm \
+    && nvm install $NODE_VERSION \
+    && ln -s `nvm which $NODE_VERSION` /usr/local/bin/node \
+    && ln -s `nvm which $NODE_VERSION` /usr/local/bin/nodejs \
+    && ln -s `dirname $(nvm which 10)`/npm /usr/local/bin/npm \
```

Comment:
should this be $NODE_VERSION instead of 10?

Reply:
YES. WHOOPS. new PR inc